x/oracle/errors.go

```diff
@@ -0,0 +1,30 @@
+package oracle
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Local code type
+type CodeType = sdk.CodeType
+
+//Exported code type numbers
+const (
+	DefaultCodespace sdk.CodespaceType = "oracle"
+
+	CodeInvalidNonce  CodeType = 1
+	CodeNotFound      CodeType = 2
+	CodeMinimumTooLow CodeType = 3

```

Comment:
Could we update the `CodeMinimumTooLow` variable name to be more reflective of the error? Could lead to misunderstanding of the expected error, as the transaction value could be too low, or the validator's available coins could be too low, etc.